docs/_reference/RTMClient.md

```diff
@@ -140,14 +139,16 @@ Disconnects from the RTM API.
 
 **Kind**: instance method of <code>[RTMClient](#RTMClient)</code>  
 
-| Param |
-| --- |
-| optReason | 
-| optCode | 
+| Param | Type |
+| --- | --- |
+| optReason | <code>Error</code> | 
+| optCode | <code>Number</code> | 
 
 <a name="RTMClient+reconnect"></a>
 
 ### rtmClient.reconnect()
+Attempts to reconnect to the websocket by retrying [rtm.start](rtm.start).
```

Comment:
maybe not be so specific, since it could be an `rtm.connect()` instead of `rtm.start()`, right?